tests/Integration/ResourceStatusSystem/Test_ResourceStatus.py

```diff
@@ -63,8 +63,7 @@ def test_addAndRemove(self):
     self.assertTrue(res['OK'])
     self.assertEqual(res['Value'][0][0], 'TestName1234')
     self.assertEqual(res['Value'][0][1], 'statusType')
-    self.assertEqual(res['Value'][0][2], 'all')
-    self.assertEqual(res['Value'][0][3], 'Active')
+    self.assertEqual(res['Value'][0][2], 'Active')
```

Comment:
You may add a test to for the `vo` too, but check that it is at the end :)

Reply:
I'll do it, but let see what Andrei says first.

Reply:
Looks fine, extra test would be appropriate

Reply:
Done this.